test/search/api_search_test.cpp

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2006-2025 Knut Reinert & Freie Universität Berlin
+// SPDX-FileCopyrightText: 2016-2025 Knut Reinert & MPI für molekulare Genetik
+// SPDX-License-Identifier: CC0-1.0
+
+#include <gtest/gtest.h>
+
+#include "../app_test.hpp"
+#include <search/search.hpp>
+
+// To prevent issues when running multiple API tests in parallel, give each API test unique names:
+struct api_search_test : public app_test
+{};
+
+TEST_F(api_search_test, default_config)
+{
+    configuration config{};
+    config.reads = data("test_reads.fasta");
+    config.index_file = data("test_index.bin");
+    config.kmer_size = 20;
+    config.threshold = 1;
+
+    testing::internal::CaptureStdout();
+    testing::internal::CaptureStderr();
+
+    search(config);
```

Comment:
```suggestion
    EXPECT_NO_THROW(search(config));
```

then you also get the error message.
Currently, `search` throws because `test_reads.fasta` does not exist (there is only a `reads.fasta` in `data`).
Because we caputre stdout and stderr, we don't see the throw message.

With `EXPECT_NO_THROW`, it continues the test and shows stderr stdout